ts-packages/web/src/app/teams/[username]/_components/team-sidemenu.tsx

```
@@ -14,22 +15,107 @@
+            />
+          ) : (
+            <div className="w-20 h-20 rounded-full border border-neutral-500 bg-neutral-600" />
+          )}
+        </div>
+
+        <div className="font-medium">{user.nickname}</div>
+
+        <div
+          id="user-profile-description"
+          className="text-xs text-gray-400"
+          dangerouslySetInnerHTML={{ __html: user.html_contents }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)